R-TypeGame/components/Velocity/VelocityComponent.hpp

```diff
@@ -0,0 +1,28 @@
+/*
+** EPITECH PROJECT, 2024
+** R-Type game
+** File description:
+** VelocityComponent
+*/
+
+#pragma once
+
+#include "IComponent.hpp"
+
+/**
+ * @brief Component Velocity for R-Type Game.
+ *
+ */
+class VelocityComponent : public AComponent {
+
+    public:
+
+        int velocity;  // Velocity.
```

Comment:
Can we have a negative speed ?

Reply:
I made some research and yes, speed can't be negative, but velocity can :). My bad